app/src/main/java/com/yalta/viewmodel/MapViewModel.kt

```diff
@@ -85,6 +87,7 @@ class MapViewModel @Inject constructor(
 
     private fun updatePoints(points: List<RoutePoint>) = viewModelScope.launch(Dispatchers.Main) {
         _points.value = points
+        storage.update(_currentRoute!!)
```

Comment:
So you just update it one way, ok for now, but in perfect implementation this VM should also get it's values from `storage`